### ISSUE TITLE
Not downloading videos

### ISSUE THREAD

**Issue by emerysteele** [Dracovian/Discord-Scraper#34]
Tried out the experimental version as the master appears to have issues going back farther than 5-6 weeks for me.

But experimental is not downloading videos.

**Comment by Dracovian** [comment]
Yes, this is likely due to the issues with embedded content. Something that's been plaguing this script for well over a year now.

I do recommend people try out the experimental version of the script as it addresses multiple problems with the master branch, but it's not all-encompassing.

So far the biggest issue I'm facing is the fact that the script grabs the whole JSON data from the server as it should, and yet less than half of the data in the JSON file is ending up being grabbed.

What that means is the script is succeeding in grabbing all of the JSON data that it's supposed to, but there are issues with grabbing the contents linked within the JSON data.

I do attempt to grab the data that Discord proxies as seen here: https://github.com/Dracovian/Discord-Scraper/blob/experimental/Discord/Discord.py#L183

I should rewrite the script again, this time around I should focus more on verbosity and error handling.

**Comment by Dracovian** [comment]
I have yet to see if the experimental_requests branch is able to grab video contents, but I am hedging a bit on 3rd party modules because it definitely seems as if I have my work cut out for me whenever it comes to maintaining my custom modules.

I also made the same recommendation in the previous conversation, the history of my response was due to not reading this conversation beforehand. So disregard my edit where I brought up the experimental branch since you already found issues with it as well.

Soon I'll merge the experimental_requests and experimental branches when everything is accounted for in the experimental_requests branch. I'll be glad to not have to type out the name of that branch when the time comes.
